ocs_ci/ocs/constants.py

```diff
@@ -1296,6 +1296,12 @@
 JAVAS3_POD_NAME = "java-s3"
 AWSCLI_SERVICE_CA_CONFIGMAP_NAME = "awscli-service-ca"
 AWSCLI_TEST_OBJ_DIR = "/test_objects/"
+TEST_NET_1_BLOCK = "192.0.2.0/24"
+TEST_NET_2_BLOCK = "198.51.100.0/24"
+TEST_NET_3_BLOCK = "203.0.113.0/24"
+TEST_NET_MCAST_BLOCK = "233.252.0.0/24"
```

Comment:
are these subnets applicable to AWS only? If so, maybe better to change the var name to reflect that